### PR TITLE
link to best practice docs

### PR DESCRIPTION
**Pull request by taylor**
Signed-off-by: Taylor Carpenter <taylor@vulk.coop>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Test Suite project](https://github.com/cncf/cnf-testsuite#cnf-conformance-test-suite) who work on the mechanics of the conformance tests.
 
-The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications.
+The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications which we are listed in:
+- [Best Practices for CNF Developers](doc/best_cnf_dev.md)
```
Comment (electrocucaracha) [inline README.md]:
```suggestion
* [Best Practices for CNF Developers](doc/best_cnf_dev.md)
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Test Suite project](https://github.com/cncf/cnf-testsuite#cnf-conformance-test-suite) who work on the mechanics of the conformance tests.
 
-The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications.
+The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications which we are listed in:
```
Comment (tomkivlin) [inline README.md]:
```suggestion
The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications which are listed in:
```

Comment (tomkivlin) [reply README.md]: Otherwise lgtm

**Review by tomkivlin — APPROVED** [review]

**Review by lixuna — APPROVED** [review]
LGTM

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**